README.md

```diff
@@ -270,7 +270,7 @@ minimal_symmetric_difference_dfa = dfa ^ other_dfa
 symmetric_difference_dfa = dfa.symmetric_difference(other_dfa, minify=False)
 ```
 
-### DFA.issubset(self, other_dfa)
+#### DFA.issubset(self, other_dfa)
```

Comment:
@Tagl Didn't even realize the hierarchy was incorrect. Thanks for fixing this. 🙂

Reply:
You're most welcome